src/dft.cpp

```diff
@@ -283,6 +283,81 @@ void dft_chunk::update_dft(double time) {
   }
 }
 
+/* Return the L2 norm of the DFTs themselves.  This is useful
+   to check whether the simulation is finished (whether all relevant fields have decayed).
+   (Collective operation.) */
+double fields::dft_norm2() {
```

Comment:
I would just call it `dft_norm` since it is not squared (unlike `fields_chunk::dft_norm2` or `dft_chunk::norm2`)

Reply:
Fixed.